guidelines/faster-review.md

```diff
@@ -0,0 +1,115 @@
+# Faster Code Review Guideline
+
+Kyma is quite a large project with many areas, each has a group of core developers
+who are working hard on developing that area of Kyma. While that means they are
+working hard on new features and enhancements, they also work on doing code
+reviews for PRs coming from different contributors from the community and it is
+crucial that the code review process of new PRs becomes quick and efficient for
+both parties. This guide defines a set of principles and practices that can help
+achieving such a goal.
+
+## 0. Familiarize yourself with project conventions
+
+- [CONTRIBUTING.md](../CONTRIBUTING.md)
+- [Naming conventions](naming.md)
+- [Content strategy](content-guidelines/content-strategy.md)
+- [Content guidelines](content-guidelines)
+- [Coding standards](coding-standards)
+
+## 1. Before coding communication
+
+We recommend these steps before you jump into coding that nice feature you want
+
+1. Make sure you have a GitHub issue created for the work you want to do that
+   - Follows one of the issues templates and make sure to describe well the
+    the changes you intend to introduce.
+   - If possible identify in the issue the relevant area label and components
+    you might touch and change.
+2. Communicate that issue upfront on the slack channel of the relevant area and
+start a discussion there to have it pre-reviewed.
+3. Ask the area group to assign you an issue buddy from the CODEOWNERS.
```

Comment:
We should somehow announce that in Kyma that we have such flow and also new "responsibility" -> `issue buddy` because when someone from external world will ask us about his/her issue buddy then our developer may be suppressed who is the issue buddy because for me it was a new term :D

Reply:
I expect that once we merge it, all org members know the doc and follow, otherwise we fail